NEWS.md

```diff
@@ -8,6 +8,8 @@
 * File name check is moved to strict_checks condition to allow underscores in the file name. Underscores are allowed in xpt but not per FDA requirements. (#126)
 * It is now possible to get and set the xportr options using the helper function `xportr_options()` (#130)
 
+* Added xportr.character_metadata_types and xportr.numeric_metadata_types so that all R types, including dates, are handled by xportr_type. In case the R type is different from the metadata type, the variable is coerced (#161).
```

Comment:
Can we add the complete log of the changes?
I think apart from the additions of these two new options two older options were modified.
So, I feel we can mention that `xportr.character_types` and `xportr.numeric_types` were changed too.

Reply:
I added that `xportr.character_types` and `xportr.numeric_types` changed in the description